internal/pkg/fixtures/objects.go

```diff
@@ -90,11 +90,11 @@ func (MockedManifest) SortKey(_ string) string {
 	return "key"
 }
 
-func (MockedManifest) GetObjectPath() string {
+func (MockedManifest) GetRelativePath() string {
```

Comment:
Premenovane `ObjectPath` -> `RelativePath`, kedze struktura `AbsPath` sa pouziva aj na ine ucely, ako su objekty, a potom to vyzera dost divne.